project_development_tips.md

```diff
@@ -2,115 +2,215 @@
-        - On the Forum or project blog
+        - Examples of places to share are your project's chat, forum, blog, email list, etc.
+        - This will allow the community and other stakeholders to have plenty of time
+        to view the upcoming changes and provide feedback if neccessary.
     - Share the time of the deployment and the deployment transaction with branch/commit hash information to ensure the transaction itself is correct.
-    - Coordinate deployment with managers to make sure it is done correctly and on time.
+    - Coordinate deployment with stakeholders to make sure it is done correctly and on time.
+
+## Responsibilities to the Community
+
+### Project should have thorough Documentation
+
+Each project should have a Detailed README.md with these sections:
+    - Explanation of the project itself with links to the app
+    - Addresses on various networks
+    - High-level technical description of the contracts with emphasis on important types and functionality
+    - Architecture diagram (if applicable)
+    - Include links to tutorials if they are external
+
+Additionally, each contract, transaction, and script should have high-level descriptions
+at the top of their files. This way, anyone in the community can easily 
+come in and understand what each one is doing without having to parse confusing code.
 
-### **Community Engagement**
+### Projects should Engage with and respond to their own Community
 
 Once a contract is deployed, the work doesn’t stop there. As the developer of a public project on a public blockchain, the owners have an obligation to be helpful and responsive to the community so that they can encourage composability and third party interactions.
 
 - Keep issues open in the repo.
 - The owner should turn on email notifications for new issue creation in the repo.
 - Respond to issues quickly and clean up unimportant ones.
-- Maybe write a blog post about some part of the contracts every once in a while
+- Maybe write a blog post about some part of the contracts every once in a while.
+
+### Projects should contribute to the greater Flow and Cadence Community
+
+Responsible project maintainers should contribute to discussions
+about important proposals (new cadence features, standard smart contracts, metadata, etc)
+and generally be aware about evolving best practices and anti-pattern understandings.
+Projects who contribute to these discussions are able to influence them to ensure
+that the language/protocol changes are favorable to them
+and the rest of the app developers in the ecosystem.
+It also helps the owner to promote the project and themselves.
+
+Owners should also regularly re-evaluate the code of their project to see if it needs to be updated 
+to conform to new standards and new versions of Cadence.
+This will help them identify potential problems with their smart contracts ahead of time
+and be able to stay on the cutting edge of smart contract development.
```

Comment:
Not sure that this is especially relevant in this section, feels out of place here.